src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,164 @@
+---
+path: "/docs/managers/mq-manager"
+title: "MQ Manager"
+---
+
+**Alpha**
+
+## Overview
+This Manager provides the ability to connect to an existing Queue Manager, enabling applications to read and write one or more messages to and from a queue. <br><br> 
+
+
+## Annotations
+
+The following annotations are available with the CICS TS CECI Manager
+<details>
+<summary>MQ Manager</summary>
+
+| Annotation: |MQ Manager |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @QueueManager |
+| Description: | The <code>@QueueManager</code> annotation represents the name of the MQ Queue Manager |
+| Syntax: | @QueueManager<br> public IMessageQueueManager qmgr;<br> |
+| Notes: | The <code>IMessageQueueManager</code> interface enables connection to the MQ Queue Manager.  |
+</details>
+
+<details>
+<summary>MQ Queue</summary>
+
+| Annotation: |MQ Queue |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @Queue |
+| Description: | The <code>@Queue</code> annotation represents the name of the MQ Queue |
+| Syntax: | @Queue<br> public IMessageQueue queue;<br> |
+| Notes: | The <code>IMessageQueue</code> interface enables the test to put the provided messages onto the queue and retrieve messages from the queue. |
+</details>
+
+
+
+## Code snippets
+
+Use the following code snippets to help you get started with the MQ Manager.
+ 
+<details><summary>Create a MQ Manager</summary>
+
+The following snippet shows the code that is required to create an instance of a MQ Manager:
+
+```java
+@QueueManager
+public IMessageQueueManager qmgr;
+```
+</details>
+
+<details><summary>Instantiate a MQ Queue</summary>
+
+The following snippet shows the code that is required to instantiate a MQ Queue:
+
+```java
+@Queue(archive = "true", name = "GALASA.INPUT.QUEUE")
+public IMessageQueue queue;
+```
+
+You can just as simply instantiate multiple MQ Queues:
+
+```java
+@Queue(archive = "true", name = "GALASA.INPUT.QUEUE")
+public IMessageQueue queue;
+	
+@Queue(archive = "false", name = "GALASA.INPUT.QUEUE2")
+public IMessageQueue queue2;
+	
+@Queue(tag = "NEWQUEUE")
+public IMessageQueue queue3;
+```
+</details>
+
+<details><summary>Put messages to a queue</summary>
+
+The following snippet shows the code required to put a message on a queue:
+
+```java
+TextMessage tm = qmgr.createTextMessage(testData);
+queue.sendMessage(tm);
+```
+</details>
+
+<details><summary>Read messages from a queue</summary>
+
+The following snippet shows the code required to read a message from a queue:
+
+```java
+Message m = queue.getMessage();
+String response = m.getBody(String.class);
```

Comment:
I need to check this code example as if this is the current api I can make that better

Reply:
no that is correct and will stay like that - sorry